### PR TITLE
Fix CPU usage report in process.GetOne

### LINKED ISSUE #6063: Beats monitoring reports 0 for its own CPU usage
Beats monitoring reports 0 for its own CPU usage.

```
  "monitoring": {
    "metrics": {
      "beat": {
        "cpu": {
          "total": {
            "norm": {
              "pct": 0
            },
            "pct": 0,
            "value": 0
          }
        },
        "info": {
          "ephemeral_id": "caf4bf11-75f3-4280-a2c6-035ac6a0f28f",
```

The `last` value is always nil because `ProcsMap` is never updated by `GetOne()`.

https://github.com/elastic/beats/blob/4d0ff2be5199f88173da48afc90adba2387d5433/libbeat/metric/system/process/process.go#L437-L438

Additionally, using a global to store the last collected CPU counters is problematic because the value is shared by all monitoring reporter types (logs and ES). This could cause the differential CPU values to be calculated incorrectly.

https://github.com/elastic/beats/blob/a3970eec1d6d0766fdfa14104f5355b5cefbabb4/libbeat/cmd/instance/metrics.go#L21-L25

### PR DESCRIPTION
**Pull request by kvch**
I added the missing line of updating the state.
However, it still produces 0 from time to time. Reporting CPU usage has to be done differently as @andrewkroh  and @urso pointed it out.

I am opening the PR, because at least the total CPU usage since start in `beat.cpu.total. value` is correct. So calculating CPU usage percentage can be done in different applications.

### CODE REVIEW

**Comment by kvch** [comment]
I am removing the label `in progress`, so it can be merged asap. I am going to open a separate PR with the separated snapshotting and reporting changes.

**Comment by kvch** [comment]
I have removed all calculated metrics including
* `beat.cpu.total.pct`
* `beat.cpu.total.norm.pct`
* `system.cpu.total.pct`
* `system.cpu.total.norm.pct`

So it would not confuse users when looking at metrics.

All metrics are going to be reimplemented.

File: libbeat/metric/system/process/process.go [hunk 1]
```diff
@@ -402,6 +402,7 @@ func (procStats *Stats) GetOne(pid int) (common.MapStr, error) {
 	}
 
 	e := procStats.getProcessEvent(p)
+	procStats.ProcsMap = newProcs
```
Comment (exekias) [inline libbeat/metric/system/process/process.go]: I don't fully get why is this updated, could you explain a bit about it? sorry, :innocent:

Comment (kvch) [reply libbeat/metric/system/process/process.go]: It contains the last samples. It must be updated so based on the previous and the current samples averages and percentages can be calculated.

**Pull request merged**